### ISSUE TITLE
Sfpdfview load from local storage iOS is not showing as it should in second time open

### ISSUE THREAD

**Issue by febryanadip96** [syncfusion/xamarin-demos#14]
I try to open pdf file from local storage in iOS using SfPDFViewer for Xamarin Forms. For the first time, it looks fine. But, in second time i open my app, It is not showing as it should and it is like zooming automatically when i open it. I had try to zoom out but it cannot.

**Comment by Gayathri-github7** [comment]
Hi febryanadip96 ,

We have tried to reproduce the issue “PDF Viewer displayed with improper zooming while loading the PDF document from local storage”, but it is working fine as expected. Kindly refer the sample in the below link which we have created to reproduce the reported issue.  
https://www.syncfusion.com/downloads/support/directtrac/246275/ze/LoadToPdfViewer-1774878061 
   
Kindly share the following details to analyze more on this issue and it will be helpful for us to provide solution at the earliest.  
1.       Modify / Simple sample with which the issue could be reproduced.  
2.       Replication procedure to reproduce the issue or screen shot illustrating the issue.  
3.       iOS device specification with the OS version.  
4.       PDF document with which the issue could be reproduced.

With Regards,
Gayathri R
(I work for Syncfusion)
